tests/DCPS/Messenger_Annotation/Args.h

```diff
@@ -0,0 +1,103 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#ifndef MESSENGER_TEST_ARGS_H
```

Comment:
Do these guards need to match file name / paths? It seems a shame to duplicate all this code (I'm assuming it's pretty similar) just for the sake of testing some different IDL compiler options. It would have been nice if  we have figured out a way to optionally change how the normal messenger test pulled in its IDL & type support.